src/resolvers/user_management.spec.js

```diff
@@ -1,9 +1,9 @@
-import Factory from '../seed/factories'
-import { GraphQLClient, request } from 'graphql-request'
-import jwt from 'jsonwebtoken'
-import { host, login } from '../jest/helpers'
+import Factory from "../seed/factories";
```

Comment:
Please use single quotes and no ending semicolon like defined in our linting.

Reply:
@Kachulio1 Run `yarn run lint --fix`

Reply:
👍